libs/platform/ebpf_object.c

```diff
@@ -11,12 +11,12 @@ static ebpf_lock_t _ebpf_object_tracking_list_lock = {0};
 /**
  * @brief Objects are added to the ID table when they are initialized and removed
  * from the table when they are freed. Objects in the table always have a
```

Comment:
Not true anymore. Please fix this comment,

Reply:
Not sure which specific issue you were referring to, so expanded and added more details.  Will follow-up to make sure your concerns were addressed.